chrome/content/zotero/xpcom/utilities.js

```diff
@@ -455,11 +455,15 @@ Zotero.Utilities = {
 	 *   <ZOTEROBREAK/> => <br/>
 	 *   <ZOTEROHELLIP/> => &#8230;
 	 *
-	 * @param {String} str
+	 * @param {String|Number} str
 	 * @return {String}
 	 */
 	"htmlSpecialChars":function(str) {
-		if (str !== undefined && typeof str != 'string') {
+		if ((str || str == 0) && typeof str != 'string') {
+			if (typeof str != 'number') {
+				Zotero.debug('#htmlSpecialChars: non-string and non-number arguments are deprecated. Update your code', 
```

Comment:
I've been looking into why this was here in the first place and it used to throw [before](https://github.com/rmzelle/zotero/blame/916db3fd4878c1e72f530c21ecb92bd0159a70ba/chrome/content/zotero/xpcom/utilities.js#L307-L309), but got changed [here](https://github.com/rmzelle/zotero/blame/48810f23e04229ace477713a4023ac3ccad49283/chrome/content/zotero/xpcom/utilities.js#L421). I regret ever touching this. Will update soon.